packages/core/admin/admin/src/pages/MarketplacePage/components/SortSelect/index.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { Select, Option } from '@strapi/design-system/Select';
+import { Typography } from '@strapi/design-system/Typography';
+import { Box } from '@strapi/design-system/Box';
+import { useIntl } from 'react-intl';
+import PropTypes from 'prop-types';
+import { getTrad } from '../../../../content-manager/utils';
+
+const SelectTypography = () => {
+  const { formatMessage } = useIntl();
+
+  return (
+    <Typography variant="pi" fontWeight="bold">
+      {formatMessage({
+        id: getTrad('admin.pages.MarketPlacePage.sort.sortBy'),
+        defaultMessage: 'Sort by',
+      })}
+    </Typography>
+  );
+};
```

Comment:
ℹ️ Not sure this is the correct way to handle this. I had a warning saying the component shouldn't be created as a prop and it should be moved out of the Parent component